test/integration/banListTest.ts

```diff
@@ -323,7 +322,7 @@ describe('Test: unbaning entities via the PolicyList.', function() {
         await moderator.setUserPowerLevel(await mjolnir.client.getUserId(), banListId, 100);
         await moderator.sendStateEvent(banListId, 'org.matrix.mjolnir.shortcode', '', { shortcode: "unban-test" });
         await mjolnir.client.joinRoom(banListId);
-        this.mjolnir!.watchList(Permalinks.forRoom(banListId));
+        await mjolnir.watchList(Permalinks.forRoom(banListId));
```

Comment:
good catch; i wonder if we can improve linting on tests to warn us when we've forgotten to await something

Reply:
Yeah, it is interesting. I don't know about this. I know that in VSCode it highlights the line with grey in this case, but it does require that it knows the type and it couldn't here because it was taking the mjolnir from `this` which there is only mocha interface information for.

Reply:
which is also why i changed that too